src/jmcomic/jm_feature.py

```
@@ -39,23 +39,21 @@ def jm_task_context(self) -> dict:
         """Return the current invocation's isolated task-context snapshot."""
         return get_jm_task_context()
 
-    def should_invoke(self, feature_from: str, when: str) -> bool:
+    def should_invoke(self, when: str) -> bool:
         """
-        判断在当前钩子(when)下，根据来源(feature_from)，是否应该执行。
+        判断在当前钩子(when)下是否应该执行。
         默认返回 True（任何钩子都执行）。子类可覆写来限制执行时机。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)